api/src/test/resources/patient_without_openmrs_id.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="windows-1250"?>
+d<?xml version="1.0" encoding="windows-1250"?>
```

Comment:
Is this file being loaded by any test? I would expect it to result into xml parsing exceptions.

Reply:
deleted "d" typo